### PR TITLE
Reconsume latest MicroProfile Config 2.0 snapshot

### PR DESCRIPTION
**Pull request by tevans78**
#build 
part of #13644

### CODE REVIEW

File: dev/io.openliberty.io.smallrye.config/src/io/openliberty/microprofile/config/internal/extension/OLSmallRyeConfigProviderResolver.java [hunk 1]
```diff
@@ -41,23 +38,23 @@ public void activate(ComponentContext cc) {
...
+//        SmallRyeConfigBuilder builder = null;
+//        final SecurityManager sm = System.getSecurityManager();
+//        if (sm != null) {
+//            builder = AccessController.doPrivileged((PrivilegedAction<SmallRyeConfigBuilder>) OLSmallRyeConfigBuilder::new);
+//        } else {
+//            builder = new OLSmallRyeConfigBuilder();
+//        }
+//
+//        return builder;
+//    }
```
Comment (Azquelt) [inline dev/io.openliberty.io.smallrye.config/src/io/openliberty/microprofile/config/internal/extension/OLSmallRyeConfigProviderResolver.java:58]: Can this be removed?

Comment (tevans78) [reply dev/io.openliberty.io.smallrye.config/src/io/openliberty/microprofile/config/internal/extension/OLSmallRyeConfigProviderResolver.java:58]: yes

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
#build

File: dev/com.ibm.ws.microprofile.config.1.4_fat/fat/src/com/ibm/ws/microprofile/config14/test/Config14Tests.java [hunk 2]
```diff
@@ -79,13 +83,21 @@ public static void setUp() throws Exception {
     }
 
     @Test
-    public void testBadObserver() throws Exception {
+    @SkipForRepeat(RepeatConfig20EE8.ID)
```
Comment (benjamin-confino) [inline dev/com.ibm.ws.microprofile.config.1.4_fat/fat/src/com/ibm/ws/microprofile/config14/test/Config14Tests.java:86]: Maybe add comments to this class' skip for repeat too?

File: dev/com.ibm.ws.microprofile.config_repeat_tests/src/com/ibm/ws/microprofile/config/fat/repeat/RepeatConfigActions.java [hunk 3]
```diff
@@ -43,7 +43,7 @@ public static FeatureReplacementAction getAction(Version version, String server)
             case CONFIG20_EE8:
                 return new RepeatConfig20EE8(server);
             case LATEST:
-                return new RepeatConfig14EE8(server); //currently the latest version is 1.4 (EE8). Will update to 2.0 when that reaches a beta standard.
+                return new RepeatConfig20EE8(server); //currently the latest version is 1.4 (EE8). Will update to 2.0 when that reaches a beta standard.
```
Comment (benjamin-confino) [inline dev/com.ibm.ws.microprofile.config_repeat_tests/src/com/ibm/ws/microprofile/config/fat/repeat/RepeatConfigActions.java:46]: Comment is out of date.

**Review by benjamin-confino — APPROVED** [review]

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
#run-libby-bot

**Comment by wraschke** [comment]
@tevans78 I believe this PR's changes are behind hard breaks in a LAOS continuous build. Please see https://ibm-cloud.slack.com/archives/C30NGUQ9E/p1600121380174400 for more details.

**Pull request merged**